modular/blocksyncer/blocksyncer_indexer.go

```diff
@@ -1,551 +0,0 @@
-	return nil
-}
-
-// ExportAccounts accepts a slice of transactions and persists accounts inside the database.
-// An error is returned if write fails.
-func (i *Impl) ExportAccounts(block *coretypes.ResultBlock, txs []*types.Tx) error {
-	return nil
-}
-
-// ExportEvents accepts a slice of transactions and get events in order to save in database.
-func (i *Impl) ExportEvents(ctx context.Context, block *coretypes.ResultBlock, events *coretypes.ResultBlockResults) error {
-	// get all events in order from the txs within the block
-	for _, tx := range events.TxsResults {
-		// handle all events contained inside the transaction
-		// call the event handlers
-		for _, event := range tx.Events {
-			if err := i.HandleEvent(ctx, block, common.Hash{}, sdk.Event(event)); err != nil {
-				return err
-			}
-		}
-	}
-	return nil
-}
-
-type TxHashEvent struct {
-	Event  sdk.Event
-	TxHash common.Hash
-}
-
-// ExportEventsInTxs accepts a slice of events in tx in order to save in database.
-func (i *Impl) ExportEventsInTxs(ctx context.Context, block *coretypes.ResultBlock, txs []*types.Tx) error {
-	bucketEvent := make([]TxHashEvent, 0)
-	groupEvent := make([]TxHashEvent, 0)
-	permissionEvent := make([]TxHashEvent, 0)
-	spEvent := make([]TxHashEvent, 0)
-	prefixEvent := make([]TxHashEvent, 0)
-	exitEvent := make([]TxHashEvent, 0)
-	virtualGroupEvent := make([]TxHashEvent, 0)
-	objectEvent := make([]TxHashEvent, 0)
-	objectIDEvent := make([]TxHashEvent, 0)
-	dataList := make([]interface{}, 0)
-
-	for _, tx := range txs {
-		txHash := common.HexToHash(tx.TxHash)
-		for _, event := range tx.Events {
-			e := TxHashEvent{Event: sdk.Event(event), TxHash: txHash}
-			if bucket.BucketEvents[event.Type] {
-				bucketEvent = append(bucketEvent, e)
-			} else if group.GroupEvents[event.Type] {
-				groupEvent = append(groupEvent, e)
-			} else if permission.PolicyEvents[event.Type] {
-				permissionEvent = append(permissionEvent, e)
-			} else if storageprovider.StorageProviderEvents[event.Type] {
-				spEvent = append(spEvent, e)
-			} else if virtualgroup.VirtualGroupEvents[event.Type] {
-				virtualGroupEvent = append(virtualGroupEvent, e)
-			} else if spExit.SpExitEvents[event.Type] {
-				exitEvent = append(exitEvent, e)
-			} else if event.Type == object.EventDeleteObject || event.Type == object.EventCreateObject || payment.PaymentEvents[event.Type] {
-				data, err := i.ExtractEvent(ctx, block, common.Hash{}, sdk.Event(event))
-				if err != nil {
-					return err
-				}
-				dataList = append(dataList, data)
-				prefixEvent = append(prefixEvent, e)
-				objectIDEvent = append(objectIDEvent, e)
-			} else if object.ObjectEvents[event.Type] {
-				objectEvent = append(objectEvent, e)
-			}
-			if prefixtree.BuildPrefixTreeEvents[event.Type] {
-				prefixEvent = append(prefixEvent, e)
-			}
-		}
-	}
-
-	if len(dataList) > 0 {
-		err := i.BatchHandle(ctx, dataList, block, common.Hash{})
```

Comment:
`BatchHandle` is a bad function name, as the following will invoke `concurrenceHandleEvent`, I did not get the difference between them from the function name.

Reply:
We guarantee the sequence of events in each module, the concurrency is just a different module concurrency, and this code is no longer available